test/test_datasets_samplers.py

```diff
@@ -38,55 +37,55 @@ def get_list_of_videos(num_videos=5, sizes=None, fps=None):
         yield names
 
 
-@unittest.skipIf(not io.video._av_available(), "this test requires av")
-class Tester(unittest.TestCase):
+@pytest.mark.skipif(not io.video._av_available(), reason="this test requires av")
+class TestDatasetsSamplers:
     def test_random_clip_sampler(self):
         with get_list_of_videos(num_videos=3, sizes=[25, 25, 25]) as video_list:
             video_clips = VideoClips(video_list, 5, 5)
             sampler = RandomClipSampler(video_clips, 3)
-            self.assertEqual(len(sampler), 3 * 3)
+            assert len(sampler) == 3 * 3
             indices = torch.tensor(list(iter(sampler)))
             videos = torch.div(indices, 5, rounding_mode='floor')
             v_idxs, count = torch.unique(videos, return_counts=True)
-            assert_equal(v_idxs, torch.tensor([0, 1, 2]))
-            assert_equal(count, torch.tensor([3, 3, 3]))
+            assert v_idxs == torch.tensor([0, 1, 2])
+            assert count == torch.tensor([3, 3, 3])
```

Comment:
Again, here and everywhere else, please do not change the existing `assert_equal` calls.

Reply:
Oh, I am sorry, @NicolasHug , for the inconvinience. Actually, I got confused. I will correct them.